### DISCUSSION TITLE
pycbc.waveform.waveform.get_td_waveform

### DISCUSSION THREAD

**Discussion by hibiki-iwanaga** [gwastro/pycbc#4851] [Q&A]
Hello,
When generating a waveform using pycbc.waveform.waveform.get_td_waveform as shown below, are the values for mass, spin, and other parameters that are directly specified given in the source frame or the detector frame?

hp, hc = get_td_waveform(approximant="SEOBNRv4_opt",
                                 mass1=m1,
                                 mass2=m2,
                                 spin1z=s1,
                                 spin2z=s2,
                                 delta_t=dt,
                                 f_lower=f_low,
                                 distance=d)

**Comment by ahnitz** [comment]
@hibiki-iwanaga Parameters are always in the detector frame.
